examples/parallel_cancel.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
Do you intend to add this to the repo, or was it just for testing?

Reply:
Whoops hadn't checked the diff yet, good catch